packages/dd-trace/src/telemetry/index.js

```diff
@@ -126,7 +127,7 @@ function getInstallSignature (config) {
 function appStarted (config) {
   const app = {
     products: getProducts(config),
-    configuration: flatten(config)
+    configuration: configWithOrigin || flatten(config)
```

Comment:
I might be misunderstanding this, but don't these 2 return different structures entirely?

Reply:
And if they don't, shouldn't the variable be set instead of skipping it?

Reply:
Actually I'm not sure I understand how it could be missing as it should be available on config instantiation.